### ISSUE TITLE
0.5.3/0.5.4 textpattern/lib/txplib_publish.php:426 twttr(): Undefined variable: out

### ISSUE THREAD

**Issue by stampedev** [cara-tm/pat_article_social#3]
Lately I'm getting an the following error, although afaik I did not change anything. Older tweets are getting displayed correctly, but that may be due to caching. Any ideas? TXP is 4.5.7, I already tried with all other plugins disabled...

```
Tag error: <txp:twttr media="0" related="0" status="https://twitter.com/stamp3de/status/671324941460066304" /> ->  Notice: Undefined variable: out while parsing form None on page default
textpattern/lib/txplib_publish.php:426 twttr()
textpattern/lib/txplib_publish.php:326 processTags()
textpattern/publish.php:547 parse()
index.php:83 textpattern()
```

Besides, thanks for this great plugin :+1:

**Comment by stampedev** [comment]
Works when specifying the numeric status (e.g. "671324941460066304") only, so might be related to URL parsing?

**Comment by cara-tm** [comment]
Hi.
Thanks for reporting and for using pat-article-social ;)
Could you tell me which version is concerning?
I'll make some tests and eventually some corrections.

**Comment by cara-tm** [comment]
Issue closed with version 0.5.5.

**Comment by cara-tm** [comment]
Sorry. Not closed.

Your problem is identified and comes from the 3 number into your Twitter account.

Corrected in the upcoming version 0.5.6.

**Comment by cara-tm** [comment]
Sorry. Not closed.

Your problem is identified and comes from the 3 number into your Twitter account.

Corrected in the upcoming version 0.5.6.
